README.md

```diff
@@ -218,6 +218,30 @@ streamingClient.sendAudioData(ByteString);
 
 The streaming connection will close when you call the method `streamingClient.close()` or if you go 15 seconds without sending any audio data.
 
+## Custom Vocabulary (for streaming)
+
+You can now submit any custom vocabularies independently through the new CustomVocabularies client! The main benefit is that users of the SDK can now submit their custom vocabularies for pre-processing and then include these processed custom vocabularies in their streaming jobs.
+
+Below you can see an example of how to create and submit as well as check on the status and other associated information of your submitted custom vocabulary!
+
+```
+// Initialize your client with your rev.ai access token
+String accessToken = "Your Access Token";
+CustomVocabulariesClient customVocabulariesClient = new CustomVocabulariesClient(accessToken);
+
+// Construct a CustomVocabulary object using your desired phrases
```

Comment:
Should you mention the limits on how many phrases you can have in a CV?

Reply:
I think we are not mentioning the things like limits on purpose to prevent having to update the documentation in too many places.